mobile/src/screens/meditation-list/libs/constants.ts

```diff
@@ -1,10 +1,3 @@
-const mockedData = [
-  { id: 0, title: '1Meditation for deep sleep', duration: 10 },
-  { id: 1, title: 'Breathing meditation', duration: 100 },
-  { id: 2, title: '2Meditation for deep sleep', duration: 99 },
-  { id: 3, title: '3Meditation for deep sleep', duration: 10 },
-  { id: 4, title: '4Breathing meditation', duration: 100 },
-  { id: 5, title: '5Meditation for deep sleep', duration: 99 },
-];
+const MOCKED_DURATION = 10;
```

Comment:
Should we use it for now?

Reply:
@v0ldemar01 i didnt see duration of track in meditation entry object for now

Reply:
@v0ldemar01 i have troubles with duration in react native track player it doesnt work properly, and i need to link another library, but how to link to our project ?

Reply:
@iamAlinaaa , What's wrong with duration in this library? Ask me, I'll help you